packages/forms-web-app/src/controllers/lpa-dashboard/confirm-add-user.js

```diff
@@ -27,6 +27,7 @@ const postConfirmAddUser = async (req, res) => {
 		});
 	}
 
+	delete req.session.addUserEmailAddress;
```

Comment:
not sure why this has been added back in - it needs to be removed so success box logic works in add-remove-users controller

Reply:
It's not something I was working on. I think it came in when I rebased from main? I can only trace it in this branch to the squash so maybe squashing also squashes commits made from the rebase?

Reply:
possibly your main wasn't up to date when rebasing?